### PR TITLE
Use consistent naming scheme for message params

### LINKED ISSUE #290: Define a naming scheme for recipient addresses and sent data
## Description

There are instances in the code where we send some data to a recipient address, and we therefore have many Key-Value structs to support these.

```rust
pub struct AddressWithAmount {
    address: String,
    amount: u64,
    return_address: Option<String>,
    expiration: Option<u32>,
}

pub struct AddressNativeTokens {
    pub address: String,
    pub native_tokens: Vec<(TokenId, U256)>,
    pub return_address: Option<String>,
    pub expiration: Option<u32>,
}

pub struct AddressAndNftId {
    pub address: String,
    pub nft_id: NftId,
}
```
This leads to fields with long, awkward names that still do not adequately describe their usage:
```rust
addresses_and_nft_ids: Vec<AddressAndNftId>,
```

I propose that we define a keyword for these Key-Value-like structs to unify them.

### Suggestions

- Package (ex. `BaseTokenPackage`, `NftPackage`)
- Parcel (same as above, but perhaps more associated with mail and sending)
- Packet
- Bundle

Please comment with any other suggestions you have.

### PR DESCRIPTION
**Pull request by DaughterOfMars**
# Description of change

Defined `Params` naming scheme for Send methods.

## Links to any relevant issues

- Closes #290

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: bindings/nodejs/lib/wallet/Account.ts [hunk 1]
```diff
@@ -716,21 +712,21 @@ export class Account {
...
+        mintNativeTokenParams: MintNativeTokenParams,
         transactionOptions?: TransactionOptions,
     ): Promise<MintTokenTransaction> {
         const response = await this.methodHandler.callAccountMethod(
             this.meta.index,
             {
                 name: 'mintNativeToken',
                 data: {
-                    nativeTokenOptions: nativeTokenOptions,
+                    mintNativeTokenParams: mintNativeTokenParams,
```
Comment (Thoralf-M) [inline bindings/nodejs/lib/wallet/Account.ts]: Shouldn't the field be only `params`? Also for the others below

Comment (DaughterOfMars) [reply bindings/nodejs/lib/wallet/Account.ts]: Yes, thank you. It's hard to find all the locations 😤

File: bindings/python/iota_sdk/types/common.py [hunk 2]
```diff
@@ -52,9 +52,9 @@ def as_dict(self):
         return config
 
 
-class AddressWithAmount():
+class SendAmountParams():
     def __init__(self, address, amount):
-        """Initialise an AddressWithAmount
+        """Initialise a SendAmountParams
```
Comment (Thoralf-M) [inline bindings/python/iota_sdk/types/common.py]:
```suggestion
        """Initialise SendAmountParams
```

File: .changes/send-params.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+"iota-sdk-bindings-core": patch
+"iota-sdk-nodejs": patch
+"iota-sdk-python": patch
```
Comment (Thoralf-M) [inline .changes/send-params.md]: We only need this for the wallet nodejs binding we release with covector
```suggestion
"wallet-nodejs-binding": patch
```

Comment (DaughterOfMars) [reply .changes/send-params.md]: Can you explain?

Comment (Thoralf-M) [reply .changes/send-params.md]: The .change files are used for covector so it knows what change should lead to which version bump and also for the changelog entry
But supported packages need to be configured here https://github.com/iotaledger/iota-sdk/blob/develop/.changes/config.json and we only have the wallet nodejs binding there (we want to get rid of covector alltogether very soon, then we will just do the changelogs as for the Rust crate and don't need the .change files anymore)

File: sdk/src/wallet/account/operations/transaction/high_level/send_nft.rs [hunk 4]
```diff
@@ -13,10 +13,10 @@ use crate::{
     wallet::account::{operations::transaction::Transaction, Account, TransactionOptions},
 };
 
-/// Address and nft for `send_nft()`
+/// Params `send_nft()`
```
Comment (Thoralf-M) [inline sdk/src/wallet/account/operations/transaction/high_level/send_nft.rs]:
```suggestion
/// Params for `send_nft()`
```
Or order changed?

File: sdk/src/wallet/bindings/nodejs/types/bridge/account.ts [hunk 5]
```diff
@@ -215,39 +214,38 @@ export type __IncreaseNativeTokenSupplyMethod__ = {
...
+        params: MintNftParams[];
         options?: TransactionOptions;
     };
 };
 
 export type __PrepareOutputMethod__ = {
     name: 'prepareOutput';
     data: {
-        options: OutputOptions;
+        options: OutputParams;
```
Comment (Thoralf-M) [inline sdk/src/wallet/bindings/nodejs/types/bridge/account.ts]:
```suggestion
        params: OutputParams;
```

File: .changes/send-params.md (hunk 3 above)
Comment (DaughterOfMars) [reply .changes/send-params.md]: Oh. Must have changed since I last looked at it. Thanks.

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**